### ISSUE TITLE
python scripts/translate_images.py PATH_TO_TRAINED_MODEL -n 100 raise RuntimeError

### ISSUE THREAD

**Issue by sonwe1e** [LS4GAN/uvcgan#25]
It raise RuntimeError when I run `python ./uvcgan-main/scripts/translate_images.py ./uvcgan-main/outdir/selfie2anime/model_d\(cyclegan\)_m\(cyclegan\)_d\(basic\)_g\(vit-unet\)_cyclegan_vit-unet-12-self-lsgan-paper-cycle_high-256/ -n 100`
![image](https://github.com/LS4GAN/uvcgan/assets/72198365/8680227a-513d-4b79-b91d-8157ae24c467)
And I found the shape of X and shape of embed is not the same
![image](https://github.com/LS4GAN/uvcgan/assets/72198365/8063850f-9280-4284-baf8-5f414983b2b2)
The position of error is
`    def forward(self, x):
        # x     : (N, L, input_features)
        # embed : (1, height * width, embed_features)
        #       = (1, L, embed_features)
        embed = self.embed(self.y_const, self.x_const)
        print(x.shape)
        print(embed.shape)

        # embed : (1, L, embed_features)
        #      -> (N, L, embed_features)
        embed = embed.expand((x.shape[0], *embed.shape[1:]))
        print(x.shape)
        print(embed.shape)

        # result : (N, L, embed_features + input_features)
        result = torch.cat([embed, x], dim=2)

        # (N, L, features)
        return self.output(result)`

**Comment by usert5432** [comment]
Hi @sonwe1e, I think this error is caused by working on images of larger size. From the error message, it looks like the model was trained on images of size (256, 256) pixels, but it is evaluated on images of size (512, 512) pixels.

Would my assumption on images sizes be correct?

**Comment by sonwe1e** [comment]
Hi! Thanks for your reply. I found there is no resize in valid dataset, so I add resize augment to valid config to match the image size.
